package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "lapi-rewriter",
+  "version": "1.0.0",
+  "description": "Rewriter presubmit script for LAPIs",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "dependencies": {
+    "jest": "^24.8.0",
```

Comment:
I think jest is a dev dependency?

Reply:
This was also autogenerated, I didn't install it as --save-dev though, so maybe I should go back and do that.